src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -822,7 +822,7 @@ def ONNXConvTransposeOp:ONNX_Op<"ConvTranspose",
   "output_shape can also be explicitly specified in which case pads values are auto generated using these equations:"
   ""
   "  total_padding[i] = stride[i] * (input_size[i] - 1) + output_padding[i] + ((kernel_shape[i] - 1) * dilations[i] + 1) - output_shape[i]"
-  "  If (auto_pads != SAME_UPPER): pads[start_i] = total_padding[i]/2; pads[end_i] = total_padding[i] - (total_padding[i]/2)"
```

Comment:
This file was generated by running: python3 gen_onnx_mlir.py (after listing CategoryMapper in the list of operators that have a custom `verify` function).  The various diffs likely indicate that the file was not generated correctly at some point in the past?